### ISSUE TITLE
Licensing

### ISSUE THREAD

**Issue by davidkarlsen** [legsem/legstar-core2#18]
Would you consider changing the license to ASF2 as it's more business friendly and would allow for more widespread use (and thus contribution)?

**Comment by davidkarlsen** [comment]
@legsem Ping?

**Comment by legsem** [comment]
The choice of AGPL for LegStar 2 is deliberate. My experience with LegStar
1 (which is LGPL, a business friendly license), is that most users are
happy to use the software for free without ever contributing anything back.

**Comment by davidkarlsen** [comment]
Hm - I would put it the other way around - with a business unfriendly license I would not be able to use it - and thus not contribute back - which I would happily do with a business friendly license. A bit like this case: https://github.com/oshoukry/openpojo/issues/63

**Comment by jcustenborder** [comment]
@legsem I have to agree with @davidkarlsen. Seeing all the other stuff you are working on, I would have thought this was licensed Apache like Avro and Hadoop.

**Comment by jcustenborder** [comment]
@legsem Have you considered dual licensing this with something like Apache 2.0?
